src/locales/en.json

```diff
@@ -431,27 +431,53 @@
+  "nft_claim_confirmation_title": "Your eBooks will be added to the following decentralized identity",
   "nft_claim_error_message_unknown": "An unknown error occurs when adding to bookshelf",
-  "nft_claim_login_in": "Verify wallet to claim your NFT",
-  "nft_claim_login_in_button": "Verify Wallet",
+  "nft_claim_error_message": "Error occurs when adding to bookshelf {error}",
+  "nft_claim_login_button_loading": "Loading your decentralized identity",
+  "nft_claim_login_button_sign_in": "Or log in with another method",
+  "nft_claim_login_button_sign_up": "Sign Up",
+  "nft_claim_login_content_collection": "交代 supporting author, have temperature and connection",
+  "nft_claim_login_content_community": "交代 being in a bigger community and foster reading",
+  "nft_claim_login_content_identity": "交代 the use and purpose of Decentralized Identity",
+  "nft_claim_login_content": "Please login to your Liker Land account to read and manage your purchased ebook.",
+  "nft_claim_login_title_collection": "Embark on your collection journey",
+  "nft_claim_login_title_community": "Participate in the community",
+  "nft_claim_login_title_identity_check": "You can also visit the 'Account' section and check this identity address on your 'Dashboard' at any time",
+  "nft_claim_login_title_identity": "Establish your decentralized identity",
+  "nft_claim_login_title": "Create Liker Land Account",
+  "nft_claim_message_input_label": "You can write anything to the author, such as support, thanks, or simply say hello. Your message will be written in the book and become a part of it.",
+  "nft_claim_message_title": "Would you like to leave a message for the author?",
+  "nft_claim_message_placeholder": "Example: Thank you for your heartfelt creation.",
   "nft_claim_missing_qs": "Kindly locate the notification email and proceed to click the provided link in order to claim your Writing NFT.",
-  "nft_claim_physical_only": "Payment successful. We will arrange delivery shortly. Thank you for your support!",
+  "nft_claim_NFT_author": "Author",
+  "nft_claim_NFT_name": "\"{name}\"",
+  "nft_claim_NFT_publisher": "Publisher",
+  "nft_claim_welcome_button_claim": "Start claiming my eBook",
+  "nft_claim_welcome_text_physical_only": "Payment successful. We will arrange delivery shortly. Thank you for your support!",
+  "nft_claim_welcome_text_send_gift": "Your eBook gift has been sent via email. You will be notified by email when the recipient claims the eBook.\nThank you for sharing the joy of reading.",
+  "nft_claim_welcome_text": "A confirmation email has been sent to your email address as well. \n\nPlease begin the ebook claiming process to receive the book from the author.\n\nYour book will be recorded on the blockchain network, in which you will have complete ownership & access to the content.",

```

Comment:
Those `\n` should not be rendered unless we apply `whitespace-pre-wrap` class

Reply:
[already applied](https://github.com/likecoin/liker-land/pull/1588/files#diff-3ad5d4692f894ca8fc663ec45fcc3531aa9cd771545dab84ca79d69b8170f4d6R18)